src/NKeys/CredentialsParser.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Basis\Nats\NKeys;
+
+/**
+ * Parser for Credentials File containing JWT and NKey seed
+ */
+class CredentialsParser
+{
+    /**
+     * @param string $text
+     * @return array{nkey?: string, jwt?: string}
+     */
+    public function parse(string $text): array
+    {
+        $jwtMatches = [];
+        $nkeyMatches = [];
+
+        if(
```

Comment:
Looks like formatting issue. Basically, we add space between if and condition

Reply:
Fixed